yard/__init__.py

```diff
@@ -1,4 +1,4 @@
 from importlib.metadata import version
 
 __version__ = version("datatractor-yard")
-__api_version__ = __version__
+__api_version__ = ".".join(__version__.split(".")[0:2])
```

Comment:
I am wondering if we shouldn't use an actual semver parser library here (e.g. `packaging.version.Version`) instead of caveman splitting.